From f25cc19e4d4e6642d46817624aa1c298179de678 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Fri, 31 Oct 2025 15:45:13 +0100
Subject: [PATCH 032/133] std.Target: bump linux max version to 6.17
X-Developer-Signature: v=1; a=openpgp-sha256; l=781; i=xtex@astrafall.org;
 h=from:subject; bh=5QkoQKM7KVGexocfr17wNpNJdHARHd4F92+ycfKDsE4=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn7cv9eqqy8/+HD6qAfEQ/NmFZMnjuRN9U48dLFgk79h
 4c2/3A90DGRhUGMi8FSTJGlyLDBm1UnnV90WbkszBxWJpAh0iINDAwMDCwMfLmJeaVGOkZ6ptqG
 eoaGOkY6RgxcnAIw1QcWMzJcFhf9srDkdLP0Pc83qbdfz/5Uv6BLfmqxRaPj/qArv4R7GRnWRTJ
 M3FW+9n/EWSaf/tkdUTtD3fgOB3ta6TNzcc8LXccDAA==
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit a9c648c06040f0fe066b1b0538137b003fdc53fd)
---
 lib/std/Target.zig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/std/Target.zig b/lib/std/Target.zig
index 2b56408e506e..c4819f58759a 100644
--- a/lib/std/Target.zig
+++ b/lib/std/Target.zig
@@ -447,7 +447,7 @@ pub const Os = struct {
 
                                 break :blk default_min;
                             },
-                            .max = .{ .major = 6, .minor = 16, .patch = 0 },
+                            .max = .{ .major = 6, .minor = 17, .patch = 0 },
                         },
                         .glibc = blk: {
                             // For 32-bit targets that traditionally used 32-bit time, we require
-- 
2.52.0

